drivers/clock_control/clock_control_smartbond.c

```diff
@@ -126,6 +127,55 @@ static void smartbond_start_xtal32k(void)
 	}
 }
 
+#ifdef CONFIG_REGULATOR
+#define SMARTBOND_CLOCK_SET_PLL_ON smartbond_clock_set_pll_status(true)
+#define SMARTBOND_CLOCK_SET_PLL_OFF smartbond_clock_set_pll_status(false)
```

Comment:
change to static functions instead of macros - it looks really weird when macro is actually a function call

Reply:
Changed